### ISSUE TITLE
 Can m3u8 work properly on the web? 

### ISSUE THREAD

**Issue by AnonymHK** [zezo357/flutter_meedu_videoplayer#59]
Can't m3u8 be used on the web? I want to use this component on Android, IOS, and the Web, but I see a prompt on the example indicating that it cannot be used under WEB!

**Comment by zezo357** [comment]
Yes, for now the web implementation doesn't allow playing hls, will try to add it in next release

**Comment by AnonymHK** [comment]
Thank you very much for your reply. Using m3u8 on the web is a very common requirement, and I look forward to adding this feature in the next version. I will try to replace "better_video_player" with your component.

**Comment by AnonymHK** [comment]
A suggestion, I think you should add a TikTok slide up and down playback example and put it at the top of the example, and add TikTok and other logo guides in the introduction of pub.dev. This will increase the number of developers trying to use it, because when I chat in the "better_video_player" developer group, most people enter by watching its scrolling examples But now 'better_video_player' is no longer maintained, and many issues cannot be resolved.

**Comment by zezo357** [comment]
>  A suggestion, I think you should add a TikTok slide up and down playback example and put it at the top of the example, and add TikTok and other logo guides in the introduction of pub.dev. This will increase the number of developers trying to use it, because when I chat in the "better_video_player" developer group, most people enter by watching its scrolling examples But now 'better_video_player' is no longer maintained, and many issues cannot be resolved.

The list view example is the closest to it, since It caches videos, but will try to improve it to look like TikTok (no promises I only work on package in free time)

**Comment by zezo357** [comment]
for m3u8 web support

Add to `pubspec.yaml`

> ```
>     video_player_web_hls:
>       git: https://github.com/zezo357/video_player_web_hls
>   ```
> 

Add  

> ```javascript
>    <script src="https://cdn.jsdelivr.net/npm/hls.js@latest"  type="application/javascript"></script>
>    ```
>    
>    in index.html above   
>   ```javascript
>    <script src="main.dart.js" type="application/javascript"></script>
>   ```
>   or
>   ```javascript
>    <script src="flutter.js" defer></script>
>   ```

can you test and let me know? i will not publish it, until video_player_web_hls accepts my pull request

**Comment by AnonymHK** [comment]
video_player_web_hls has accepted Pull and released a new version!

**Comment by zezo357** [comment]
Yes will update the package readme,

The only thing changing is instead of using git in pubspec


You will use the version
